common/supplemental/supplementalData.xml

```diff
@@ -1890,7 +1890,7 @@ XXX Code for transations where no currency is involved
 		<language type="lv" scripts="Latn" territories="LV"/>
 		<language type="lwl" scripts="Thai"/>
 		<language type="lzh" scripts="Hans" alt="secondary"/>
-		<language type="lzz" scripts="Latn Geor"/>
+		<language type="lzz" scripts="Geor Latn"/>
```

Comment:
Is the ordering significant?

Reply:
The code is writing it alphabetically, so when I re-generate the script it force-alphabetizes it. There is an argument it should be ordered by usage --- however the XML is just not a good way to capture this because the labelling is unclear.

Reply:
The scripts (and regions) should be in ranked order, not sorted. If the code is sorting them, that's a bug.

Reply:
It looks like if Roozbehs' items are taken care of, then this would be ready to merge into 47.

Reply:
I'm glad Roozbeh took a look ;) I resolved this by changing the non-Latn script for these languages to be considered "secondary" in language_script.tsv

Merging this changes ended up getting really messy so I'll post a new pull request.